util/file/fileutil.go

```diff
@@ -9,7 +9,17 @@ import (
 	"io/ioutil"
 	"strings"
 
+	"github.com/klauspost/pgzip"
 	log "github.com/sirupsen/logrus"
+	"k8s.io/utils/env"
+)
+
+var gzipImpl = env.GetString(GZipImplEnvVarKey, PGZIP)
```

Comment:
I did not know about this library - neat

Reply:
Yea it's super fast and works great for large workflows.